jerry-port/default/default-debugger-transport.c

```diff
@@ -13,15 +13,15 @@
  * limitations under the License.
  */
 
-#include "debugger.h"
-#include "jcontext.h"
 #include "jerryscript-port.h"
+#include "jmem.h"
 
 #ifdef JERRY_DEBUGGER
 
 #include <arpa/inet.h>
 #include <errno.h>
 #include <fcntl.h>
+#include <string.h>
```

Comment:
After this PR, I would split this file up such that you have one platform-agnostic (shared) file and one with the (Linux) platform specific stuff separated out into a different file. That way you can re-use the shared file as-is instead of having to copy-paste so much code in a project that uses TCP/websockets but doesn't have a Linux-y API (such as Zephyr).

Reply:
Yeah, that makes sense